src/pages/SearchMobilePage/SearchMobilePage.js

```diff
@@ -1,21 +1,108 @@
-import React from 'react'
+import React, { useState } from 'react'
+import cx from 'classnames'
+import { Link } from 'react-router-dom'
+import Tabs from '@santiment-network/ui/Tabs'
+import Icon from '@santiment-network/ui/Icon'
 import MobileHeader from './../../components/MobileHeader/MobileHeader'
 import Search from './../../components/Search/SearchContainer'
+import {
+  getRecentAssets,
+  getRecentTrends,
+  removeRecentTrends,
+  removeRecentAssets
+} from '../../utils/recent'
+import allProjects from '../../allProjects.json'
 import styles from './SearchMobilePage.module.scss'
 
-const SearchMobilePage = ({ history }) => (
-  <MobileHeader
-    goBack={history.goBack}
-    backRoute={'/'}
-    classes={{
-      wrapper: styles.wrapper,
-      right: styles.hidden,
-      title: styles.hidden
-    }}
-    title='Search'
-  >
-    <Search className={styles.search} />
-  </MobileHeader>
-)
+export const TABS = [
+  {
+    index: 'Assets',
+    content: 'Assets'
+  },
+  {
+    index: 'Trends',
+    content: 'Trends'
+  }
+]
+
+const SearchMobilePage = ({ history }) => {
+  const [selectedTab, selectTab] = useState(TABS[0].index)
+  const onSelectTab = selected => selectTab(selected)
+  const assetsRecent = getRecentAssets()
+  const assetsWithInfo = assetsRecent.map(asset => ({
+    ...allProjects.find(({ slug }) => slug === asset)
```

Comment:
If the user opens the project which not contains in saved json, then we can't show it in recent. It's ok now, but we need to save this as a card

Reply:
I think, better to remove this logic now and use only slugs in recent block